src/scripts/OSFramework/OSUI/ErrorCodes.ts

```diff
@@ -52,13 +46,8 @@ namespace OSFramework.OSUI.ErrorCodes {
 		FailToSetTargetElement: 'OSUI-GEN-06001',
 	};
 
-	export const Submenu = {
-		FailRegisterCallback: 'OSUI-GEN-14001',
-	};
-
 	export const Tooltip = {
-		FailRegisterCallback: 'OSUI-GEN-08001',
-		FailOnSetIntersectionObserver: 'OSUI-GEN-08002',
+		FailOnSetIntersectionObserver: 'OSUI-GEN-08001',
```

Comment:
I don't agree with this. the old number should be commented and this stay the same.